app/app.js

```diff
@@ -75,7 +85,7 @@ window.run = function() {
                     errorControllerPromise
                 );
 
-            var layoutSetupPromise = Promise.join(
+            return Promise.join(
```

Comment:
There is a timing issue when setting up tabs which occurs sometimes. Can we change the tabBar setup back to be similar to the setup in this commit? https://github.com/mobify/astro-scaffold/blob/0bbd75817f7d0b6fd83560660b48d83c460c5715/app/app.js